qemu/tests/cfg/cpu_device_hotpluggable.cfg

```diff
@@ -1,9 +1,11 @@
 # Notes:
 #    For ppc64/ppc64le, please manually specify cpu_model in your environment
 - cpu_device_hotpluggable: install setup image_copy unattended_install.cdrom
-    only ppc64 ppc64le x86_64
+    no aarch64
     Windows:
-        no WinXP, WinVista, Win7, Win8, Win10, Win2000, Win2003
+        only Win2008, Win2012, Win2016, Win2019
```

Comment:
Would you please elaborate about this change? thanks.

Reply:
@luckyh Feature owners want to make a more clearer supported list of win guests.
https://github.com/autotest/tp-qemu/pull/2176#discussion_r404580044

Reply:
@PaulYuuu I see, but the side effect is we will have to add support for future windows distributions, should let them be aware of this.

Reply:
@luckyh I agree with you. So I am considering two methods:
- Add a "#TBD" comment above supported list
- Revert this line
Which one do you think is better?

Reply:
@PaulYuuu both looks good to me, you can have a discussion with them, thanks.

Reply:
@luckyh updated, thanks for pointing out.
CC'ing @huangyum

Reply:
Looks good to me.